framework/components/ADatePicker/ADatePicker.js

```diff
@@ -156,9 +189,12 @@ ADatePicker.propTypes = {
    */
   onChange: PropTypes.func,
   /**
-   * Sets the date selected.
+   * Sets the date(s) selected.
    */
-  value: PropTypes.instanceOf(Date)
+  value: PropTypes.oneOfType([
+    PropTypes.instanceOf(Date),
+    PropTypes.array,
```

Comment:
I added a custom prop type validator for the range tuple. It checks to determine if the array passed as a range is a tuple with either `Date` objects or `null`. The reason the first date can be null is in the case of the ending date being selected, but not the starting date. `[null, new Date(someDateInFuture)`. Although my current PR with `useADateRange` doesn't make it so that this is possible, the idea is that if we do want to add that type of functionality, this prop validator will check for that.